plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2646,27 +2629,26 @@ public void testAllAvailableTypes()
 
         // SHOW STATS
         assertThat(query("SHOW STATS FOR test_all_types"))
-                .projected(0, 2, 3, 4, 5, 6) // ignore data size which is varying for Parquet (and not available for ORC)
                 .skippingTypesCheck()
                 .matches("VALUES " +
-                        "  ('a_boolean', NULL, 0.5e0, NULL, 'true', 'true'), " +
-                        "  ('an_integer', NULL, 0.5e0, NULL, '1', '1'), " +
-                        "  ('a_bigint', NULL, 0.5e0, NULL, '1', '1'), " +
-                        "  ('a_real', NULL, 0.5e0, NULL, '1.0', '1.0'), " +
-                        "  ('a_double', NULL, 0.5e0, NULL, '1.0', '1.0'), " +
-                        "  ('a_short_decimal', NULL, 0.5e0, NULL, '1.0', '1.0'), " +
-                        "  ('a_long_decimal', NULL, 0.5e0, NULL, '11.0', '11.0'), " +
-                        "  ('a_varchar', NULL, 0.5e0, NULL, NULL, NULL), " +
-                        "  ('a_varbinary', NULL, 0.5e0, NULL, NULL, NULL), " +
-                        "  ('a_date', NULL, 0.5e0, NULL, '2021-07-24', '2021-07-24'), " +
-                        "  ('a_time', NULL, 0.5e0, NULL, NULL, NULL), " +
-                        "  ('a_timestamp', NULL, 0.5e0, NULL, " + (format == ORC ? "'2021-07-24 03:43:57.987000', '2021-07-24 03:43:57.987999'" : "'2021-07-24 03:43:57.987654', '2021-07-24 03:43:57.987654'") + "), " +
-                        "  ('a_timestamptz', NULL, 0.5e0, NULL, '2021-07-24 04:43:57.987 UTC', '2021-07-24 04:43:57.987 UTC'), " +
-                        "  ('a_uuid', NULL, 0.5e0, NULL, NULL, NULL), " +
-                        "  ('a_row', NULL, " + (format == ORC ? "0.5" : "NULL") + ", NULL, NULL, NULL), " +
-                        "  ('an_array', NULL, " + (format == ORC ? "0.5" : "NULL") + ", NULL, NULL, NULL), " +
-                        "  ('a_map', NULL, " + (format == ORC ? "0.5" : "NULL") + ", NULL, NULL, NULL), " +
-                        "  (NULL, NULL, NULL, 2e0, NULL, NULL)");
+                        "  ('a_boolean', NULL, NULL, 0.5e0, NULL, 'true', 'true'), " +
+                        "  ('an_integer', NULL, NULL, 0.5e0, NULL, '1', '1'), " +
+                        "  ('a_bigint', NULL, NULL, 0.5e0, NULL, '1', '1'), " +
+                        "  ('a_real', NULL, NULL, 0.5e0, NULL, '1.0', '1.0'), " +
+                        "  ('a_double', NULL, NULL, 0.5e0, NULL, '1.0', '1.0'), " +
+                        "  ('a_short_decimal', NULL, NULL, 0.5e0, NULL, '1.0', '1.0'), " +
+                        "  ('a_long_decimal', NULL, NULL, 0.5e0, NULL, '11.0', '11.0'), " +
+                        "  ('a_varchar', " + (format == PARQUET ? "87e0" : "NULL") + ", NULL, 0.5e0, NULL, NULL, NULL), " +
```

Comment:
is it always exactly 87?

can you please run iceberg module on a CI couple times?

Reply:
Ran stress test (30 attempts) in #11727 and the value seems stable.